README.md

```diff
@@ -23,35 +23,50 @@ The jsep built files will be in the build/ directory.
 
 #### Client-side
 
-```javascript
-<script src="/PATH/TO/jsep.min.js"></script>
-...
-let parse_tree = jsep("1 + 1");
+```html
+<script type="module">
+import { Jsep } from '/PATH/TO/jsep.min.js';
+const parsed = Jsep.parse('1 + 1');
+</script>
```

Comment:
I think here we might want to show something closer to the original example, using the IIFE bundle. 
Or maybe both.

Reply:
Sounds good - I think I have it building the IIFE so it's more compatible now